test/state/host.cpp

```diff
@@ -410,20 +410,13 @@ evmc_tx_context Host::get_tx_context() const noexcept
         std::min(m_tx.max_priority_gas_price, m_tx.max_gas_price - m_block.base_fee);
     const auto effective_gas_price = m_block.base_fee + priority_gas_price;
 
-    return evmc_tx_context{
-        intx::be::store<uint256be>(effective_gas_price),  // By EIP-1559.
-        m_tx.sender,
-        m_block.coinbase,
-        m_block.number,
-        m_block.timestamp,
-        m_block.gas_limit,
+    return evmc_tx_context{intx::be::store<uint256be>(effective_gas_price),  // By EIP-1559.
+        m_tx.sender, m_block.coinbase, m_block.number, m_block.timestamp, m_block.gas_limit,
         m_block.prev_randao,
         0x01_bytes32,  // Chain ID is expected to be 1.
-        uint256be{m_block.base_fee},
-        intx::be::store<uint256be>(m_block.blob_base_fee),
-        m_tx.blob_hashes.data(),
-        m_tx.blob_hashes.size(),
-    };
+        uint256be{m_block.base_fee}, intx::be::store<uint256be>(m_block.blob_base_fee),
+        m_tx.blob_hashes.data(), m_tx.blob_hashes.size(), m_tx_initcodes.data(),
+        m_tx_initcodes.size()};
```

Comment:
This will keep the field initialization unrolled.

```suggestion
        m_tx_initcodes.size(),};
```